scene/gui/text_edit.cpp

```diff
@@ -7105,6 +7109,8 @@ void TextEdit::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::REAL, "v_scroll_speed"), "set_v_scroll_speed", "get_v_scroll_speed");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "hiding_enabled"), "set_hiding_enabled", "is_hiding_enabled");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "wrap_enabled"), "set_wrap_enabled", "is_wrap_enabled");
+	ADD_PROPERTY(PropertyInfo(Variant::INT, "scroll_vertical"), "set_v_scroll", "get_v_scroll");
```

Comment:
Shouldn't those values be floats?

Reply:
Right, you are.  A sec. You're really fast btw, it's amazing.

Reply:
Well, thanks. Also, don't forget to update the value type in the docs.

Reply:
The docs already have the correct ones as far as I can see, float for vertical and integer for horizontal scroll. Just like the methods in text_edit.cpp

Reply:
> integer for horizontal scroll

That's very strange. Well, this can be changed in a later PR if need be.

Reply:
gotcha, I committed the change and squashed the commits.